docs/en-US/component/popconfirm.md

```diff
@@ -39,18 +39,18 @@ popconfirm/trigger-event
-| confirmButtonText | Confirm button text                                                                 | String             | —               | —              |
-| cancelButtonText  | Cancel button text                                                                  | String             | —               | —              |
-| confirmButtonType | Confirm button type                                                                 | String             | —               | Primary        |
-| cancelButtonType  | Cancel button type                                                                  | String             | —               | Text           |
-| icon              | Icon Component                                                                      | String / Component | —               | QuestionFilled |
-| iconColor         | Icon color                                                                          | String             | —               | #f90           |
-| hideIcon          | is hide Icon                                                                        | Boolean            | —               | false          |
-| teleported        | whether popconfirm is teleported to the body                                        | boolean            | true / false    | true           |
-| persistent        | when popconfirm inactive and `persistent` is `false` , popconfirm will be destroyed | boolean            | —               | false          |
+| Attribute           | Description                                                                         | Type               | Accepted Values                                               | Default        |
+| ------------------- | ----------------------------------------------------------------------------------- | ------------------ | ------------------------------------------------------------- | -------------- |
+| title               | Title                                                                               | String             | —                                                             | —              |
+| confirm-button-text | Confirm button text                                                                 | String             | —                                                             | —              |
+| cancel-button-text  | Cancel button text                                                                  | String             | —                                                             | —              |
+| confirm-button-type | Confirm button type                                                                 | String             | primary / success / warning / danger / info / <del>text</del> | primary        |
+| cancel-button-type  | Cancel button type                                                                  | String             | primary / success / warning / danger / info / <del>text</del> | text           |
```

Comment:
`text` here is not a deprecated text, so please remove `<del>`

Reply:
resolve! thank you for you detail review!

Reply:
resolve! thank you for detail review!

by the way, there is an error in the test of 'time-picker' that I did not modify. 🤔

Reply:
Sometimes, CI will failed with unknown reason. (there's a problem in our unit test). Just re-run the tests.